Void/Void.Core/Commands.fs

```diff
@@ -3,10 +3,13 @@
 [<RequireQualifiedAccess>]
 type Command =
     | Noop
+    | PublishEvent of Event
+    //
+    | ChangeToMode of Mode

```

Comment:
Okay. So long as you are good with that. I don't want to shut you down on that, just make sure you understand my reasoning.